notebooks

```diff
@@ -1 +1 @@
-docs/notebooks/
\ No newline at end of file
+docs/notebooks
```

Comment:
is this needed?

Reply:
This is a link, and I didn't change it. `GitHub` shows it in a different way, don't know why.